bin/cli.js

```diff
@@ -169,11 +183,13 @@ const loadConfigMiddleware = async options => {
   if (options.config && fs.existsSync(options.config)) {
     const { config: configPath } = options;
     await loadConfig(configPath, options);
```

Comment:
This load config shouldn't need `await`

Reply:
Addressed in [24c78e2](https://github.com/HubSpot/hubspot-cli/pull/1307/commits/24c78e289fc7bf4e8832826b131a7ce821d8c462)